<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addTimerDialog</class>
 <widget class="QDialog" name="addTimerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add new timer</string>
  </property>
  <widget class="QWidget" name="DialogGrid" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#DialogGrid {
	
	background-color: rgb(113, 113, 113);

}

QLabel {
	color: rgb(238, 238, 236);
	font: bold;
	margin-left: 3px;
}

QPushButton {
	background-color: rgb(113, 113, 113);	
	border: 2px solid rgb(200,200,200);
	border-radius: 8px;
	
	font: 14pt bold;
	color: rgb(230,230,230);
}

QPushButton#createButton {
	
	background-color: rgb(213, 126, 76);
}

QPushButton#createButton:hover {
	background-color: rgb(220, 136, 86);
}

QPushButton#cancelButton:hover {
	background-color: rgb(123, 123, 123);
}

QLineEdit {
	border-radius: 10px;
	color: rgb(238, 238, 236);
	background-color: rgb(255, 204, 157,10);
}

QTimeEdit {
	border-radius: 10px;
	color: rgb(238, 238, 236);
	font: 19pt bold;
	background-color: rgb(255, 204, 157,10);
}

QTimeEdit::up-button  {
	subcontrol-origin: border;
	 subcontrol-position: top right;	

	background-color: rgb(213, 126, 76);
	width: 25px;
	height: 25px;
	border-width: 3px;
	border-color: rgb(85, 87, 83);
	border-radius: 4px;

	margin-bottom: 2px;
}

QTimeEdit::down-button  {
	subcontrol-origin: border;
	subcontrol-position: bottom right;

	background-color: rgb(213, 126, 76);
	width: 25px;
	height: 25px;
	border-width: 3px;
	border-color: rgb(85, 87, 83);
	border-radius: 4px;
	margin-top: 2px;
}

</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Interval:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="interval">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>12</family>
          <pointsize>19</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Timer name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="timerName">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>55</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="maxLength">
         <number>25</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="createButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
